### ISSUE TITLE
Makefile optimized build for Raspberry Pi

### ISSUE THREAD

**Issue by deadprogram** [hybridgroup/gocv#272]
It would be great if the Makefile could detect when installing on a Raspberry Pi, perhaps by checking

```
cat /etc/os-release | grep Raspbian
```

And then include

```
-D ENABLE_NEON=ON -D ENABLE_VFPV3=ON
```

As part of the `make build` task, as mentioned here:
https://www.pyimagesearch.com/2017/10/09/optimizing-opencv-on-the-raspberry-pi/

**Comment by deadprogram** [comment]
Some WIP on this is in https://github.com/hybridgroup/gocv/tree/feature/raspimake

**Comment by deadprogram** [comment]
This has been released as part of v0.16.0 thank you.
